src/elements/content-sidebar/activity-feed/task-new/Task.scss

```diff
@@ -40,6 +40,10 @@ $bcs-task-left-offset: 10 * $bcs-task-grid + 2px;
 .bcs-Task-status {
     margin-top: 3 * $bcs-task-grid;
     margin-bottom: 2 * $bcs-task-grid;
+
+    & > * {
+        margin-right: 4px;
```

Comment:
this is equal to $bcs-task-grid (which should now be `$bdl-grid-unit` since that has been created)

Reply:
Got it, will change